### PR TITLE
Minor editorial changes suggested by @iherman.

### LINKED ISSUE #73: Minor editorial issues
(I put them all in one issue...)

[§2.7 Nesting of Blank Nodes without Blank Node Identifiers](https://www.w3.org/TR/rdf12-turtle/#unlabeled-bnodes), first paragraph, the second occurrence of blankNodePropertyList should be typeset as `<code>`

---
[§6.4 Escape Sequences](https://www.w3.org/TR/rdf12-turtle/#sec-escapes) third escape sequence row, the first "hex" (after `\U`) is an active link to the production rule, although the relevant row is reproduced underneath. It should probably look like the other occurrences of "hex".

---
[§6.6 Selected Terminal Literal Strings](https://www.w3.org/TR/rdf12-turtle/#selected-terminal-literal-strings), towards the end of the table of U+007E: shouldn't "tilda" be "tilde"? In Webster, ["tilde"](https://www.merriam-webster.com/dictionary/tilde) is a valid term; "tilda" does not exist in it. 

Also: U+007C: shouldn't it say "Vertical bar"? Note that the term "verticle" also appears in the list of literal sequences for `{|` and `|}`.

---

[§7.3.1 Reifiers](https://www.w3.org/TR/rdf12-turtle/#reifier) the word "the" is duplicated in the second sentence.

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #73.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/76.html" title="Last updated on Oct 23, 2024, 4:11 PM UTC (2b0bd8e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/76/9f7284a...2b0bd8e.html" title="Last updated on Oct 23, 2024, 4:11 PM UTC (2b0bd8e)">Diff</a>

### CODE REVIEW

**Review by afs — APPROVED** [review]

**Review by domel — APPROVED** [review]

**Review by iherman — APPROVED** [review]
I agree with @TallTed in https://github.com/w3c/rdf-turtle/issues/73#issuecomment-2429526161 on aligning the name of `|` with the official one. Approving provided that change.

**Comment by gkellogg** [comment]
> I agree with @TallTed in [#73 (comment)](https://github.com/w3c/rdf-turtle/issues/73#issuecomment-2429526161) on aligning the name of `|` with the official one. Approving provided that change.

That change should be in this PR already.

File: spec/index.html [hunk 1]
```diff
@@ -1648,9 +1648,9 @@ <h2>Selected Terminal Literal Strings</h2>
       <dd>two concatenated circumflex accent characters, each having the code point <code class="codepoint">U+005E</code></dd>
       <dt id="cp-curly-pipe"><code>{|</code></dt>
       <dd><code title="left curly bracket">{</code> (left curly bracket, code point <code class="codepoint">U+007B</code>) followed by
-        <code title="verticle bar">|</code> (verticle bar, code point  <code class="codepoint">U+007C</code>)</dd>
+        <code title="vertical bar">|</code> (vertical bar, code point  <code class="codepoint">U+007C</code>)</dd>
```
Comment (iherman) [inline spec/index.html:1651]:
```suggestion
        <code title="vertical line">|</code> (vertical line, code point  <code class="codepoint">U+007C</code>)</dd>
```

File: spec/index.html [hunk 2]
```diff
@@ -1599,13 +1599,13 @@ <h2>Selected Terminal Literal Strings</h2>
         </tr>
         <tr id="cp-pipe">
           <td><code class="codepoint">U+007C</code></td>
-          <td><code title="verticle bar">|</code></td>
-          <td>Verticle bar</td>
+          <td><code title="vertical bar">|</code></td>
+          <td>Vertical bar</td>
```
Comment (iherman) [inline spec/index.html:1603]:
```suggestion
          <td><code title="vertical line">|</code></td>
          <td>Vertical line</td>
```

File: spec/index.html [hunk 3]
```diff
@@ -1648,9 +1648,9 @@ <h2>Selected Terminal Literal Strings</h2>
       <dd>two concatenated circumflex accent characters, each having the code point <code class="codepoint">U+005E</code></dd>
       <dt id="cp-curly-pipe"><code>{|</code></dt>
       <dd><code title="left curly bracket">{</code> (left curly bracket, code point <code class="codepoint">U+007B</code>) followed by
-        <code title="verticle bar">|</code> (verticle bar, code point  <code class="codepoint">U+007C</code>)</dd>
+        <code title="vertical bar">|</code> (vertical bar, code point  <code class="codepoint">U+007C</code>)</dd>
       <dt id="cp-pipe-curly"><code>|}</code></dt>
-      <dd><code title="verticle bar">|</code> (verticle bar, code point <code class="codepoint">U+007C</code>) followed by
+      <dd><code title="vertical bar">|</code> (vertical bar, code point <code class="codepoint">U+007C</code>) followed by
```
Comment (iherman) [inline spec/index.html:1653]:
```suggestion
      <dd><code title="vertical line">|</code> (vertical line, code point <code class="codepoint">U+007C</code>) followed by
```

**Comment by iherman** [comment]
> 
> That change should be in this PR already.

It wasn't... See my change comments.

cc @TallTed

**Pull request merged**